lib/src/database/indexeddb_box.dart

```diff
@@ -13,6 +13,8 @@ class BoxCollection with ZoneTransactionMixin {
 
   BoxCollection(this._db, this.boxNames, this.name);
 
+  static const bool isWeb = true;
```

Comment:
Could the isWeb checks be replaced by the kIsWeb implementation? if they run on a chromedriver I assume kisWeb should be true. We already do this in some voip code.

https://api.flutter.dev/flutter/foundation/kIsWeb-constant.html

Reply:
no but you can just do `bool.fromEnvironment('dart.library.js_util');`

Reply:
oh really? Wasn't aware of this. That's great! Thank you so much :)

Reply:
Have changed it :)